.circleci/config.yml

```diff
@@ -144,30 +144,36 @@ commands:
       study_key:
         type: string
     steps:
+      - run:
+          name: Setup gcloud context
+          command: |
+            set -u
+            readvault.sh secret/pepper/${ENVIRONMENT}/v1/conf .data.gcp.serviceKey | gcloud auth activate-service-account --key-file=-
+            gcloud --quiet config set compute/zone ${GOOGLE_COMPUTE_ZONE}
       - run:
           name: <<parameters.study_key>> generate config files and copy pepperConfig.js
           command: |
             set -u
-            ./build-study.sh v1 $ENVIRONMENT . <<parameters.study_key>> "$STUDY_GUID" --config
-            cd ddp-workspace
-            CONFIG_DIR="${OUTPUT_DIR}/assets/config"
-            mkdir -p $CONFIG_DIR
-            CONFIG_FILE_SRC_PATH="${ANGULAR_PROJECT_DIR_PATH}/output-config/pepperConfig.js"
-            cp  $CONFIG_FILE_SRC_PATH $CONFIG_DIR
-            echo "$CONFIG_FILE_SRC_PATH has been copied to $CONFIG_DIR"
-
-            if [[ "<<parameters.study_key>>" == "atcp" ]]; then
-              cp "${ANGULAR_PROJECT_DIR_PATH}/output-config/app.yaml" "${ANGULAR_PROJECT_DIR_PATH}/app.yaml"
+            if [[ "<<parameters.study_key>>" != "dsm-ui" ]]; then
+              ./build-study.sh v1 $ENVIRONMENT . <<parameters.study_key>> "$STUDY_GUID" --config
+              CONFIG_DIR="${OUTPUT_DIR}/assets/config"
+              mkdir -p $CONFIG_DIR
+              CONFIG_FILE_SRC_PATH="${ANGULAR_PROJECT_DIR_PATH}/output-config/pepperConfig.js"
+              cp  $CONFIG_FILE_SRC_PATH $CONFIG_DIR
+              echo "$CONFIG_FILE_SRC_PATH has been copied to $CONFIG_DIR"
+
+              if [[ "<<parameters.study_key>>" == "atcp" ]]; then
+                cp "${ANGULAR_PROJECT_DIR_PATH}/output-config/app.yaml" "${ANGULAR_PROJECT_DIR_PATH}/app.yaml"
+              fi
+            else
+              cd ddp-workspace
+              CONFIG_DIR="${OUTPUT_DIR}/assets/js"
+              mkdir -p $CONFIG_DIR
+              gcloud --project="broad-ddp-${ENVIRONMENT}" secrets versions access latest --secret="study-manager-ui-config" > "${CONFIG_DIR}/ddp_config.js"
             fi
```

Comment:
Had to use an `if` statement to deal with dsm ui because as mentioned earlier, it gets its configuration fie in a different way.